src/openloco/input/mouse_input.cpp

```diff
@@ -1464,8 +1464,23 @@ namespace openloco::input
                 break;
             }
             case ui::scrollview::scroll_part::hscrollbar_track_right:
-                call(0x4c8aa6, regs);
+            {
+                // 0x004C8AA6
+                ui::window* w = WindowManager::find(_pressedWindowType, _pressedWindowNumber);
```

Comment:
Every single one of these switch statements starts with doing this find. Move it up to before the switch then we can move all the checks to the one location.

Reply:
I think we should start refactoring after reworking #348